librenms_library.py

```diff
@@ -0,0 +1,247 @@
+#! /usr/bin/env python
```

Comment:
Can you place this in a subdirectory?  There are already some scripts in the LibreNMS directory...

Reply:
Sure, I was actually working on if I could merge with the existing LibreNMS/__init__.py code (at least for the database code which is pretty much the same), but there are way too much other imports in that file already.
Once I've sorted out what I can merge, I'll move the file :)

Reply:
Great.